server/gql/schema.js

```diff
@@ -117,11 +123,24 @@ const upsertResolver = async ({
         field
     };
 
+    let result;
     if (id) {
         const updated = await resolver.findById(id);
-        return updated.update(properties);
+        result = updated.update(properties);
+    } else {
+        result = resolver.create(properties);
     }
-    return resolver.create(properties);
+
+    publish(DEFAULT_CHANNEL, { reload: "Resolver" });
```

Comment:
@pb82 as I told you on friday, publishing on upserting a resolver was missing. I added it here.